cmake error on windows

**Describe the bug**
When using the cmake build method on windows, cmake errors with the following message:
```
CMake Error at cmake/utils/build.cmake:88 (file):
  file failed to open for reading (No such file or directory):

    C:/Users/CyrilPichard/develop/cpichard/arnold-usd/plugins/ndr/
Call Stack (most recent call first):
  plugins/ndr/CMakeLists.txt:39 (generate_plug_info_for_testsuite)


CMake Error at cmake/utils/build.cmake:88 (file):
  file failed to open for reading (No such file or directory):

    C:/Users/CyrilPichard/develop/cpichard/arnold-usd/plugins/usd_imaging/
Call Stack (most recent call first):
  plugins/usd_imaging/CMakeLists.txt:62 (generate_plug_info_for_testsuite)


CMake Warning at schemas/CMakeLists.txt:34 (message):
  usdGenSchema was not found, the schemas won't be generated with it.
  Consider using the internal usdgenschema by setting
  BUILD_USDGENSCHEMA_ARNOLD=True
```

--- a/plugins/ndr/CMakeLists.txt
+++ b/plugins/ndr/CMakeLists.txt
@@ -33,7 +33,8 @@ if (NOT BUILD_WITH_USD_STATIC)
     target_compile_definitions(ndrArnold PRIVATE "NDRARNOLD_EXPORTS=1")
 
     # Configure plugInfo.json, it only need the library extension
-    install_ndr_arnold_pluginfo(../ndrArnold "${CMAKE_CURRENT_BINARY_DIR}/plug/plugInfo.json" "${PREFIX_PLUGINS}")
+    set(NDR_PLUGINFO "${CMAKE_CURRENT_BINARY_DIR}/plug/plugInfo.json")
+    install_ndr_arnold_pluginfo(../ndrArnold "${NDR_PLUGINFO}" "${PREFIX_PLUGINS}")
 
     # We replicate the layout of the plugin installation inside the build dir for running the testsuite without installing.
     generate_plug_info_for_testsuite(TARGET_NAME ndrArnold TARGET_PLUGINFO "${NDR_PLUGINFO}")

--- a/plugins/usd_imaging/CMakeLists.txt
+++ b/plugins/usd_imaging/CMakeLists.txt
@@ -52,10 +52,10 @@ if (NOT BUILD_WITH_USD_STATIC)
     
     install(TARGETS usdImagingArnold
         DESTINATION "${PREFIX_PLUGINS}")
-
+    set(PLUGINFO "${CMAKE_CURRENT_BINARY_DIR}/plug/plugInfo.json")
     install_usdimaging_arnold_pluginfo(
             ../usdImagingArnold
-            "${CMAKE_CURRENT_BINARY_DIR}/plug/plugInfo.json"
+            "${PLUGINFO}"
             ${PREFIX_PLUGINS})
 
     # We replicate the layout of the plugin installation inside the build dir for running the testsuite without installing.

